src/connect.py

```diff
@@ -1,4 +1,3 @@
-
 import sys
```

Comment:
Looks good. It is always good to add 1-2 sentences saying the purpose of a file. Please add if you got a chance.